Src/FluentAssertions/FluentAction.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+
+namespace FluentAssertions
+{
+    /// <summary>
+    /// Contains static methods to help with exception assertions on actions.
+    /// </summary>
+    [DebuggerNonUserCode]
+    public static class FluentAction
+    {
+        /// <summary>
+        /// Invokes the specified action so that you can chain it with ShouldThrow, etc.
```

Comment:
`Should().Throw()`?
`ShouldThrow()` is pre-5.0.0 notation.

Reply:
Fixed: 03d3877c1dd239da08d2c5650493a64ede1f0d8c